src/main/resources/test_data.sql

```diff
@@ -0,0 +1,33 @@
+INSERT INTO crew_members VALUES ('1', 'Jonh', 'Smitt', 'captain of the ship', '1975-07-12', 'USA');
+INSERT INTO crew_members VALUES ('2', 'Dik', 'Cheny', 'first pilot', '1973-09-12', 'UK');
+INSERT INTO crew_members VALUES ('3', 'Dominic', 'Red', 'board conductor', '1975-02-12', 'Australia');
+INSERT INTO crew_members VALUES ('4', 'Sandra', 'Cooper', 'stewardess', '1989-09-05', 'New Zealand');
+INSERT INTO crew_members VALUES ('5', 'Didrih', 'Kolc', 'captain of the ship', '1978-03-05', 'Germany');
+INSERT INTO crew_members VALUES ('6', 'Peter', 'Stock', 'first pilot', '1979-07-30', 'USA');
+INSERT INTO crew_members VALUES ('7', 'Stiv', 'Jocker', 'board conductor', '1987-10-17', 'UK');
+INSERT INTO crew_members VALUES ('8', 'Anna', 'Bush', 'stewardess', '2000-12-09', 'Austia');
```

Comment:
I guess it could be typo - Austia -> Austria

Reply:
fixed in